### DISCUSSION TITLE
Is there a way to disable 'sticky' window edges?

### DISCUSSION THREAD

**Discussion by kantastisk** [linuxmint/discussions#338] [Questions]
In Cinnamon, windows stick to screen edges, panels and other window edges. This can make for some stuttery window management that I dislike, especially with large windows. Disabling window snapping/tiling doesn't make a difference. Is there some other way to change this behavior?

**Comment by rcalixte** [comment]
Hi, I can't seem to replicate this. When I disable the setting, windows no longer snap or tile for me, without stuttering.

![Window Tiling](https://github.com/linuxmint/discussions/assets/10281587/20d82974-2900-4cac-821d-184e742da936)

Can you describe more of the issues you're seeing and which applications experience it? Can you attach a video of the issue when it happens?

**Reply by kantastisk** [reply]
I might not be the best at describing it – not sure of the terminology. I'm talking about the 'stickiness' that applies even between window edges when they meet. It's different from tiling/snapping to screen edges, even though the stickiness also activates there. It applies to any window when dragging.

https://github.com/linuxmint/discussions/assets/16965712/3fa5fb14-a89a-41e4-aabb-39ea99c71a1e

**Reply by rcalixte** [reply]
I see. This is the window-to-window overlapping/snapping. I'm not sure if there is a setting for this.
